cjwstate/modules/i18n/catalogs/extract/code.py

```diff
@@ -23,7 +21,7 @@ def _find_messages_in_module_code(
     code_file, relative_path_name: str
 ) -> Dict[str, Dict[str, Any]]:
     messages_data = extract(
-        _extract_module_code,
+        extract_module_code,
         code_file,
         keywords={"trans": (1, 2)},
```

Comment:
What does the `(1, 2)` mean?

Reply:
Means that we are interested in the arguments `1` and `2` of `trans`

Reply:
I've grepped for `keywords.get`, `keywords.items` and `keywords[` and found nothing. I can't find any place where we read values from `keywords`. Also, pybabel's Extract `keywords` argument is not a dict.

Can't it just be an `Iterable`? (I imagine passing `Set`/`FrozenSet` would be most efficient; and pybabel would probably pass `List`....)

Reply:
Indeed, I copied it from the call to the higher-level `extract`, which expects a `dict`, but at this point a `List` is expected (and anything that supports the `in` operator would work).